CMakeLists.txt

```diff
@@ -389,8 +389,15 @@ set(OD_SOURCEFILES
     ${SRC}/utils/FrameRateLimiter.cpp
     ${SRC}/utils/Helper.cpp
     ${SRC}/utils/LogManager.cpp
-    ${SRC}/utils/LogManagerFile.cpp
-    ${SRC}/utils/LogManagerOgre.cpp
+    ${SRC}/utils/LogManager.h
+    ${SRC}/utils/LogMessageLevel.h
```

Comment:
They will automatically be added be the cpp file that include them. Look at the source list. There is no header at all there.